startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
Chaos overwrites itself, but running chaos use in-memory version of code. So we pull new code, check syntax, but if the code is incorrect - [restart does not take place](https://github.com/chaosbot/chaos/blob/master/startup.d/90-chaos.sh#L3).